lib/lighthouse/service_exception.rb

```diff
@@ -90,9 +90,8 @@ def self.transform_error_keys(error_body, status, title, detail, code)
     # log errors
     def self.send_error_logs(error, service_name, lighthouse_client_id, url)
       logging_options = { url:, lighthouse_client_id: }
-      if error.response.present?
-        # Faraday error may contain request data
-        error.response.delete(:request)
```

Comment:
After [this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/17194), request data will no longer be included in the error body by default, so we can remove this check.